packages/example-getting-started/README.md

```diff
@@ -1,6 +1,10 @@
 # @loopback/example-getting-started
 
 This is the basic tutorial for getting started with Loopback 4!
+[Click here](https://loopback.io/doc/en/lb4/Getting-started.html) to follow
```

Comment:
I'm assuming the contents in this link are going to change with another PR, since right now it's just creating a simple app using `lb app` and adding a  `hello world` controller.

Reply:
I think the plan is to replace contents of `Getting-started` with the tutorial from `todo` app